docs/basic/getting-started/function-components.md

```diff
@@ -11,14 +11,14 @@ type AppProps = {
   message: string;
 }; /* use `interface` if exporting so that consumers can extend */
 
-// Easiest way to declare a Function Component; return type is inferred.
-const App = ({ message }: AppProps) => <div>{message}</div>;
+// Easiest way to declare a Function Component that expects a prop value; return type is inferred.
+const App: React.FC<Props> = ({ message }: AppProps) => <div>{message}</div>;
 
 // you can choose annotate the return type so an error is raised if you accidentally return some other type
-const App = ({ message }: AppProps): JSX.Element => <div>{message}</div>;
+const App: React.FC<Props> = ({ message }: AppProps): JSX.Element => <div>{message}</div>;
 
 // you can also inline the type declaration; eliminates naming the prop types, but looks repetitive
-const App = ({ message }: { message: string }) => <div>{message}</div>;
+const App: React.FC<Props> = ({ message }: { message: string }) => <div>{message}</div>;
```

Comment:
this goes against the sense of `inline`. I do understand that the idea was to define the component as any other function.